<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>To-Do List с Базой Данных</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="filterLayout">
     <item>
      <widget class="QLabel" name="label_search">
       <property name="text">
        <string>Поиск:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="search_input">
       <property name="placeholderText">
        <string>Поиск по заголовку...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_status">
       <property name="text">
        <string>Статус:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="status_filter">
       <item>
        <property name="text">
         <string>Все</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ожидает</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>В процессе</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Выполнено</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table">
     <column>
      <property name="text">
       <string>Заголовок</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Описание</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Статус</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Создано</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Изменено</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_edit">
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_refresh">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>